### PR TITLE
Indicate that there is information on DoT hover

### PR DESCRIPTION
**Pull request by dawidpotocki**
Fixes https://github.com/privacytoolsIO/privacytools.io/issues/1199

It's not a question mark, because it seemed to me like we are not sure about that they support DoT or something, huh.

screenshot

* https://deploy-preview-1232--privacytools-io.netlify.com/providers/dns/#icanndns

### CODE REVIEW

**Comment by dawidpotocki** [comment]
> I think it sticks the eye maybe a bit too much, but it's OK

Well if you want I can give DoT normal font-weight and/or use
https://fontawesome.com/icons/info?style=solid.

**Comment by Mikaela** [comment]
> Looks much better but I think just a space between the DoT and icon would be consistent with the other icons next to text in the table. But otherwise looks good to me.

said @nitrohorse

File: _includes/sections/dns.html [hunk 1]
```diff
@@ -107,7 +107,7 @@ <h1 id="icanndns" class="anchor"><a href="#icanndns"><i class="fas fa-link ancho
           <span class="no-text-wrap">
             DoH,
             <span data-toggle="tooltip" data-placement="bottom" data-original-title="Supports port 443 in addition to 853">
-              <strong>DoT</strong>,
+              DoT<span class="fas fa-info-circle fa-sm text-secondary"></span>,
```
[the remark this answers is not in the document]
Comment (Mikaela) [reply _includes/sections/dns.html]: @blacklight447-ptio Do you think @dawidpotocki should be given time to address this request or that it's currently fine and we can leave it for another PR?

I would wait.

[the remark this answers is not in the document]
Comment (blacklight447-ptio) [reply _includes/sections/dns.html]: if there is more to do, then we wait.

**Pull request merged**